bftengine/cmf/execution_data.cmf

```diff
@@ -0,0 +1,22 @@
+
+Msg RequestInput 1 {
+    uint16 clientId
+    uint64 executionSequenceNum
+    string cid
+    uint64 flags
+    string request
+    string signature
+    uint64 requestSequenceNum
+}
+
+Msg Timestamp 2 {
+    int64 time_since_epoch
+    uint64 request_position
+}
+
+Msg ExecutedRequests 2 {
+    list RequestInput Requests
+    Timestamp timestamp
```

Comment:
The `Timestamp` has to be optional because the Time Service is optional itself.

Reply:
right, thanks!